File: src/main/java/co/aplicared/jvm/juego/arbúcies/level/tile/GrassTile.java
package co.aplicared.jvm.juego.arbúcies.level.tile;

import co.aplicared.jvm.juego.arbucies.graphics.Sprite;
import co.aplicared.jvm.juego.arbúcies.graphics.Screen;

public class GrassTile extends Tile {

    protected GrassTile(Sprite sprite) {
        super(sprite);
    }

    @Override
    public void render(int x, int y, Screen screen) {
        screen.renderTile(x << 4, y << 4, this);
    }
}

File: src/main/kotlin/co/aplicared/jvm/juego/arbucies/level/tile/GrassTile.kt
package co.aplicared.jvm.juego.arbucies.level.tile

import co.aplicared.jvm.juego.arbucies.graphics.Screen
import co.aplicared.jvm.juego.arbucies.graphics.Sprite

class GrassTile(sprite: Sprite) : Tile(sprite) {
    override fun render(x: Int, y: Int, screen: Screen) {
        screen.renderTile(x, y, this)
    }
}